FindOpenStudioSDK.cmake

```diff
@@ -17,37 +17,37 @@ set(OPENSTUDIO_EXT "tar.gz")
 if(APPLE)
   set(OPENSTUDIO_PLATFORM "Darwin-${ARCH}")
   if(ARCH MATCHES "arm64")
-    set(OPENSTUDIO_EXPECTED_HASH b412f1cd6ab6747c75dece567291e81e)
+    set(OPENSTUDIO_EXPECTED_HASH 389a02d7f9d3526454bca74393ac7a6e)
   else()
-    set(OPENSTUDIO_EXPECTED_HASH 00d0321aa6c4ab5a3c36fe2335a07157)
+    set(OPENSTUDIO_EXPECTED_HASH TODO_MISSING_TODO) # There is no mac intel (x86_64) installer
```

Comment:
Will there be a Mac x86 installer or is this the end for that platform?

Reply:
I am hoping there will be one, but not sure.